README.md

```diff
@@ -1,19 +1,23 @@
-# [<img src="https://avatars0.githubusercontent.com/u/11345641?s=88&v=4" alt="VRK" width="18"/> suomifi-ui-components](https://vrk-kpa.github.io/suomifi-ui-components/)
+# [<img src="https://avatars0.githubusercontent.com/u/11345641?s=88&v=4" alt="DVV" width="18"/> suomifi-ui-components](https://vrk-kpa.github.io/suomifi-ui-components/)
+
+![npm](https://img.shields.io/npm/v/suomifi-ui-components)
+
+![NPM](https://img.shields.io/npm/l/suomifi-ui-components)
+
+![WCAG](https://img.shields.io/badge/WCAG%202.1-AA-green)
```

Comment:
Badges would be great to have on the same line. Like this, by separating them only with space-character.

![npm](https://img.shields.io/npm/v/suomifi-ui-components) ![NPM](https://img.shields.io/npm/l/suomifi-ui-components) ![WCAG](https://img.shields.io/badge/WCAG%202.1-AA-green)

Reply:
Good idea. I'll do that 👍